x-pack/plugins/security_solution/public/common/components/query_bar/index.test.tsx

```diff
@@ -20,6 +20,20 @@ describe('QueryBar ', () => {
   const mockOnSubmitQuery = jest.fn();
   const mockOnSavedQuery = jest.fn();
 
+  const Proxy = (props: QueryBarComponentProps) => (
+    <TestProviders>
+      <QueryBar {...props} />
+    </TestProviders>
+  );
+
+  // The data plugin's `SearchBar` is lazy loaded, so we need to ensure it is
+  // available before we mount our component with Enzyme.
+  const getWrapper = async (Component: ReturnType<typeof Proxy>) => {
+    const { getByTestId } = render(Component);
+    await waitFor(() => getByTestId('queryInput')); // check for presence of query input
+    return mount(Component);
+  };
```

Comment:
`React.Suspense` doesn't play nice with Enzyme, but I wanted to avoid re-writing each test with `@testing-library/react` since I'm not familiar with this area of code. So I made this wrapper which ensures the `SearchBar` that's loaded with `React.lazy` is available before Enzyme `mount` is called.

You may want to consider a rewrite with `@testing-library/react` that doesn't rely on `<SearchBar />` internals, as it should speed these tests up a bit. (Or perhaps a functional test if you really want to test the end-to-end experience).